modules/manage/pages/kubernetes/scale.adoc

```diff
@@ -15,7 +15,10 @@ IMPORTANT: You cannot decrease the number of CPU cores in a running cluster.
 
 Horizontal scaling (also known as scaling out/in) means adding more brokers to the cluster, thereby distributing the load and data across different worker nodes.
 
-. If you're scaling out, make sure that you have one additional worker node for each Redpanda broker that you want to add.
+. If you're scaling out:
+- Make sure that you have one additional worker node for each Redpanda broker that you want to add.
```

Comment:
Ensure that you have one additional worker node for each Redpanda broker that you want to add.